jormungandr-lib/src/interfaces/vote/mod.rs

```diff
@@ -0,0 +1,69 @@
+use serde::Serialize;
+
+#[derive(Serialize)]
+#[serde(remote = "chain_impl_mockchain::block::BlockDate")]
+struct BlockDate {
+    pub epoch: u32,
+    pub slot_id: u32,
+}
```

Comment:
There is a `BlockDate` type here already. Though, this one is better in the sense it uses `serde(remote)`. Add a `BlockDateDef` in this module which is serialize/deserialize for serde.

https://github.com/input-output-hk/jormungandr/blob/7aef084df61a272b56c189322cb37c84aca70b38/jormungandr-lib/src/interfaces/blockdate.rs#L5-L6

Reply:
That one in that module implements serialize, deserialize already. Gonna use that.